### ISSUE TITLE
Skin changer crashing game

### ISSUE THREAD

**Issue by Fergusonnn** [degeneratehyperbola/NEPS#123]
Skin Changer is crashing game. Pls fix

**Comment by SteakySteaks** [comment]
works fine for me

**Comment by degeneratehyperbola** [comment]
Yeah, this is a known issue. It is happening because of some invalid instruction in CS:GO that gets called by the skinchanger when fetching skins. The thing is... it is invalid only for some processors, mainly older ones.

**Comment by Patrixsin** [comment]
> Yeah, this is a known issue. It is happening because of some invalid instruction in CS:GO that gets called by the skinchanger when fetching skins. The thing is... it is invalid only for some processors, mainly older ones. And this issue is pretty hard to debug.

https://github.com/notgoodusename/OsirisAndExtra/ has working skinz code, mbe try found difference to figure out, crashing on old cpus

**Comment by Xsintashi** [comment]
If u want working skin changer, u need to change Enhanced Instruction Set from Advanced Vector Extensions 2 (AVX2) to Streaming SIMD Extensions 2 (SSE2)

**Comment by 6ph1nx1s4** [comment]
You can find this fix here -> https://github.com/degeneratehyperbola/NEPS/pull/144
